dvc/command/imp.py

```diff
@@ -15,12 +13,11 @@
 class CmdImport(CmdBase):
     def run(self):
         try:
-            default_out = os.path.basename(urlparse(self.args.url).path)
-
-            out = self.args.out or default_out
-
             self.repo.imp(
-                self.args.url, out, self.args.resume, fname=self.args.file
+                self.args.url,
+                self.args.out,
+                self.args.resume,
+                fname=self.args.file,
```

Comment:
should we update the `import_parser` below to include a new supported URL? also, would be great to create a checkbox in the doc's ticket related to packages to update import command reference as well.

Reply:
Which new supported url? `dvc import` doesn't support anything new, it is API's `repo.imp()` that does.

Reply:
so, there is no way to do `dvc import` with some package, like `pkg://something/sds`?

Reply:
Yes, there is no way to do that. Use `dvc pkg import` instead.

Reply:
This is strange though that API is not aligned with the command line though. Does it make sense to reuse imp at all in this case?